plugin/fzf.vim

```diff
@@ -334,19 +336,20 @@ function! fzf#wrap(...)
 endfunction
 
 function! s:use_sh()
-  let [shell, shellslash] = [&shell, &shellslash]
+  let [shell, shellslash, shellcmdflag] = [&shell, &shellslash, &shellcmdflag]
   if s:is_win
     set shell=cmd.exe
     set noshellslash
+    let &shellcmdflag = has('nvim') ? '/s /c' : '/c'
```

Comment:
`/s` requires setting `shellxquote`. Use the default values for Windows: `"` for nvim, `(` for vim.